js/ui/overlay/ui.overlay.js

```diff
@@ -312,7 +313,6 @@ const Overlay = Widget.inherit({
         this._initInnerOverlayClass();
 
         const $element = this.$element();
-        this._$wrapper.addClass($element.attr('class'));
```

Comment:
We have a lot of failed tests in dataGrid, Scheduler and drawer, so let's retrieve this row and remove it later in a separate pull request